addon/ng2/blueprints/component-test/files/src/app/components/__name__/__name__.spec.ts

```diff
@@ -18,8 +17,8 @@ describe('<%= classifiedModuleName %> Component', () => {
   beforeEachProviders(() => []);
 
 
-  it('should ...', injectAsync([TestComponentBuilder], (tcb:TestComponentBuilder) => {
-    return tcb.createAsync(<%= classifiedModuleName %>).then((fixture) => {
+  it('should ...', inject([TestComponentBuilder], (tcb:TestComponentBuilder) => {
```

Comment:
I am not completely sure about this. Seems like the update to remove this would give false passing tests.